qemu/tests/block_commit_reboot.py

```diff
@@ -0,0 +1,46 @@
+import time
+import random
+from qemu.tests import blk_commit
+
+
+class BlockCommitReboot(blk_commit.BlockCommit):
+
+    def reboot(self):
+        """
+        Reset guest with system_reset;
+        """
+        params = self.parser_test_args()
+        method = params.get("reboot_method", "system_reset")
```

Comment:
As method actually doesn't work in below line, should these 2 lines removed? Then method keeps default value.

Reply:
@luckyh What's your opinion? Seems this part worth arguing.

Reply:
if you want to use `system_reset` as the boot method,  `boot_check=False`  can not be omited

Reply:
@hereischen @pezhang  Updated. Please help to check again.